dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -458,6 +466,7 @@ public Workflow getWorkflow(@ApiParam(hidden = true) @Auth User user,
 
         // This somehow forces users to get loaded
         Hibernate.initialize(workflow.getUsers());
+        workflow.getVersions().forEach(workflowVersion -> Hibernate.initialize(workflowVersion.getValidations()));
```

Comment:
Hmmm, we might want a new endpoint for validations.
Or at least a new endpoint/query parameter to get a workflow fully with all lazy loaded content (starred users, validations, etc.) It a bit of work but I'm wary of the performance impact of hitting all getWorkflow calls with this